contracts/BaseLaunchpeg.sol

```diff
@@ -186,27 +188,27 @@ abstract contract BaseLaunchpeg is
         __ERC2981_init();
 
         __ERC721A_init(_name, _symbol);
-        initializeBatchReveal(_batchRevealSize, _collectionSize);
+        initializeBatchReveal(
```

Comment:
Can you try to move`initializeBatchReveal` into `InitialisePhases` ? You may have to use the `BatchReveal` struct from the factory to avoid stack too deep errors.

Reply:
does this mean that we won't specify the batch reveal config when the launchpeg contract is created as well?

Reply:
Yeah imo it makes more sense to enter the reveal date along with the sale dates.